### ISSUE TITLE
Cannot deploy Flexipage with blankSpace

### ISSUE THREAD

**Issue by JiriKarpisek** [forcedotcom/cli#2893]
### Summary
We have flexipage / dynamic form with blankSpace element when retrieve the page and deploy back there is an error We couldn't retrieve the design time component information for component flexipage:blankSpace.  

### Steps To Reproduce
in flexipage within flexiPageRegions add itemInstance below and try to deploy it on scratch org
<itemInstances>
    <componentInstance>
        <componentName>flexipage:blankSpace</componentName>
        <identifier>flexipage_blankSpace2</identifier>
    </componentInstance>
</itemInstances>

### Expected result
Should be successfully deployed

### Actual result
deploy return error: We couldn't retrieve the design time component information for component flexipage:blankSpace.  

### System Information
sf project deploy start -m Flexipage:Vehicle_Inspection_Form_Record_Page --ignore-conflicts --verbose --json
{
  "status": 1,
  "result": {
    "checkOnly": false,
    "completedDate": "2024-05-30T08:50:50.000Z",
    "createdBy": "[masked]",
    "createdByName": "User User",
    "createdDate": "2024-05-30T08:50:47.000Z",
    "details": {
      "componentFailures": [
        {
          "changed": false,
          "componentType": "FlexiPage",
          "created": false,
          "createdDate": "2024-05-30T08:50:49.000Z",
          "deleted": false,
          "fileName": "flexipages/Vehicle_Inspection_Form_Record_Page.flexipage",
          "fullName": "Vehicle_Inspection_Form_Record_Page",
          "problem": "We couldn't retrieve the design time component information for component flexipage:blankSpace.",
          "problemType": "Error",
          "success": false
        }
      ],
      "componentSuccesses": [
        {
          "changed": true,
          "componentType": "",
          "created": false,
          "createdDate": "2024-05-30T08:50:49.000Z",
          "deleted": false,
          "fileName": "package.xml",
          "fullName": "package.xml",
          "success": true
        }
      ],
      "runTestResult": {
        "numFailures": 0,
        "numTestsRun": 0,
        "totalTime": 0,
        "codeCoverage": [],
        "codeCoverageWarnings": [],
        "failures": [],
        "flowCoverage": [],
        "flowCoverageWarnings": [],
        "successes": []
      }
    },
    "done": true,
    "id": "[masked]",
    "ignoreWarnings": false,
    "lastModifiedDate": "2024-05-30T08:50:50.000Z",
    "numberComponentErrors": 1,
    "numberComponentsDeployed": 0,
    "numberComponentsTotal": 1,
    "numberTestErrors": 0,
    "numberTestsCompleted": 0,
    "numberTestsTotal": 0,
    "rollbackOnError": true,
    "runTestsEnabled": false,
    "startDate": "2024-05-30T08:50:47.000Z",
    "status": "Failed",
    "success": false,
    "files": [
      {
        "fullName": "Vehicle_Inspection_Form_Record_Page",
        "type": "FlexiPage",
        "state": "Failed",
        "problemType": "Error",
        "filePath": "/Users/[masked]/Salesforce/[masked]/force-app/main/default/flexipages/Vehicle_Inspection_Form_Record_Page.flexipage-meta.xml",
        "error": "We couldn't retrieve the design time component information for component flexipage:blankSpace."
      }
    ]
  },
  "warnings": []
}

**Comment by cristiand391** [comment]
@JiriKarpisek check the comments above you need to provide more CLI info.

FWIW, I tested this in a record page with dynamic forms by adding new `Blank space` component in lightning app builder, retrieved with `sf project retrieve start` and deployed again with `sf project deploy start -m FlexiPage:Property_Record_Page` successfully. 
Can you repro this by adding the blank space in lightning app builder and **not** modifying the files locally?

Notes:
the new `Blank space` component for dynamic forms was added in the Summer 24 release, your org should be on v61.0 to support it (if you can't see it in the `Fields` tab under `Field components` then that might explain it).
![Screenshot 2024-05-30 at 3 21 53 PM](https://github.com/forcedotcom/cli/assets/6853656/3d79fc71-d280-4e80-8bf0-7acd4f8bb209)

**Comment by JiriKarpisek** [comment]
Sorry for late answer, this is my CLI version
```
sf version --verbose --json
{
  "architecture": "darwin-arm64",
  "cliVersion": "@salesforce/cli/2.44.8",
  "nodeVersion": "node-v20.13.1",
  "osVersion": "Darwin 23.5.0",
  "rootPath": "/Users/jiri.karpisek/.local/share/sf/client/2.44.8-c60512e",
  "shell": "zsh",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 3.0.18 (core)",
    "@oclif/plugin-commands 3.3.4 (core)",
    "@oclif/plugin-help 6.0.22 (core)",
    "@oclif/plugin-not-found 3.1.10 (core)",
    "@oclif/plugin-plugins 5.1.3 (core)",
    "@oclif/plugin-search 1.0.24 (core)",
    "@oclif/plugin-update 4.2.14 (core)",
    "@oclif/plugin-version 2.1.2 (core)",
    "@oclif/plugin-warn-if-update-available 3.0.19 (core)",
    "@oclif/plugin-which 3.1.10 (core)",
    "@salesforce/cli 2.44.8 (core)",
    "apex 3.1.13 (core)",
    "auth 3.6.17 (core)",
    "data 3.4.3 (core)",
    "deploy-retrieve 3.8.3 (core)",
    "info 3.3.3 (core)",
    "limits 3.3.9 (core)",
    "marketplace 1.2.8 (core)",
    "org 4.1.14 (core)",
    "packaging 1.27.0 (user)",
    "schema 3.3.9 (core)",
    "settings 2.2.7 (core)",
    "sobject 1.4.6 (core)",
    "source 3.3.18 (core)",
    "telemetry 3.3.11 (core)",
    "templates 56.2.9 (core)",
    "trust 3.7.3 (core)",
    "user 3.5.11 (core)",
    "sfdmu 4.33.17 (user)"
  ]
}
```
@cristiand391 I retrieved the page from an org where admin prepared the changes and is failing the validation against scratch org. I did not make any manual changes in XML file.
Still getting error "We couldn't retrieve the design time component information for component flexipage:blankSpace"

Blank space is added to page 5 times and example is below
![image](https://github.com/forcedotcom/cli/assets/150060142/f652fafd-fa6e-4ae8-a71a-c09de16df36c)

**Comment by JiriKarpisek** [comment]
Oh sorry I missed your comment, the issue is version, I thought it is from spring24 so have been using v.60.0 org, so in 4 days it should resolved, I am sorry for wasting your time.
